docker/Dockerfile.rhel8

```diff
@@ -82,6 +82,8 @@ COPY ./rhsm-ca /etc/rhsm/ca
 RUN rm -f /etc/rhsm-host && \
     microdnf update -y && \
     microdnf install -y \
+      python3 \
+      python3-pip \
```

Comment:
@rdefosse We don't need pip lets remove that package and install directly via  python3 and python3-jinja2. I did the same for RHEL and Ubuntu it works fine.

Reply:
My method does not require any subscription. And `python3-pip` is included when installing `python3`. Just making sure.

Reply:
Okay